docs/modules/clusters/pages/ucn-enable.adoc

```diff
@@ -0,0 +1,70 @@
+= Enable {ucn}
+:description: For security reasons, you must enable {ucn} using static configuration. You can do this programmatically, or using the Hazelcast configuration file.
+:page-enterprise: true
+:page-beta: false
+
+{description}
+
+NOTE: If using dynamic persistence, this must be enabled in the Hazelcast configuration file. For further information persisting changes, see the xref:configuration:dynamic-config.adoc#manual[Dynamic Configuration] Overview topic under the Configure and Manage Clusters section.
+
+== Enable Programmatically
+
+To enable {ucn} programmatically, create a _Member.java_ file, add the following, then save the file to your configuration folder:
+
+[source,jave]
+----
+package com.hazelcast.namespace.staticconfig.classconfig;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.config.UserCodeNamespaceConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+HazelcastInstance hz = Hazelcast.newHazelcastInstance(config);
+
+public class Member {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+        config.getNamespacesConfig().setEnabled(true);
+    }
+}
+----
+
+== Enable in Configuration File
+
+To enable {ucn} using the _hazelcast_ configuration file, complete the following steps:
+
+. Open the _hazelcast_ configuration file in your preferred editor
++
+The _hazelcast_ configuration file can be either an XML or YAML file. 
```

Comment:
```suggestion
. Open the _hazelcast_ configuration file in your preferred editor (configuration file can be either XML or YAML) 
```

Reply:
Quick rewrite to avoid the sentence without period being followed by a sentence with a period, though I note there's a few instances of this in these pages. Another one for the style guide discussions but will park for now...

Reply:
Yes, style guide discussion. 

If I am editing existing info I use the same formatting as existing items. 

When writing from scratch, habits kick in. We do not punctuate lists grammatically - most people don't read the punctuation, so no punctuation in a list does not impede comprehension. Another in-built editor habit, I'm afraid